Tabloid/client/src/components/ApplicationViews.js

```diff
@@ -18,29 +19,33 @@ export default function ApplicationViews({ isLoggedIn, role }) {
           <Route path="login" element={<Login />} />
           <Route path="register" element={<Register />} />
           <Route path="tags" >
-          <Route index
-          element={
-            isLoggedIn
-              ? role === "Admin"
-                ? <Tags />
-                : <Navigate to="/tags" />
-              : <Navigate to="/login" />
-          }
-          />
+            <Route index
+              element={
+                isLoggedIn
```

Comment:
It seems like this still has the nested ternary in it. Can this use the same logic as the ListCategories route below it?

Reply:
I just fixed it.